apis/pipeline/pipeline.yaml

```diff
@@ -2662,6 +2662,31 @@ paths:
                 default:
                     $ref: '#/components/responses/Error'
 
+    /api/v1/orgs/{orgId}/clusters/{id}/update:
+        parameters:
+            - $ref: '#/components/parameters/orgId'
+            - $ref: '#/components/parameters/clusterId'
+
+        put:
+            security:
+                - bearerAuth: []
+            tags:
+                - cluster-update
+            summary: Update a cluster
+            description: Update a cluster
+            operationId: ClusterUpdate
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/ClusterUpdateRequest'
+            responses:
+                202:
+                    description: Accepted
+                default:
+                    $ref: '#/components/responses/Error'
```

Comment:
**Recommended**: we **SHOULD** update the the `409 Conflict` `error` and `message` here, because it refers to the secret by default.
https://github.com/banzaicloud/pipeline/pull/3123/files#diff-2b6bfc5b5d040867ca5e5f7b1df191f1R11868-R11869

Also we **SHOULD** update the `403 Access Denied` `message` at the default values, because it seems to be misphrased `You are unauthorized to perform this action access resource.`.